ssh/ssh.go

```diff
@@ -0,0 +1,50 @@
+package ssh
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/ONSdigital/dp-cli/aws"
+	"github.com/ONSdigital/dp-cli/config"
+	"github.com/ONSdigital/dp-cli/out"
+	"github.com/pkg/errors"
+)
+
+// Open an ssh connect to the specified environment
+func Launch(cfg *config.Config, env config.Environment, instance aws.EC2Result) error {
+	if len(cfg.SSHConfig.User) == 0 {
+		return errors.New("DP_SSH_USER variable must be set")
```

Comment:
I looks like this error message is a carry over from the old tool.

```suggestion
		return errors.New("missing ssh user in config file")
```